deploy/olm-catalog/kogito-operator/manifests/kogito-operator.clusterserviceversion.yaml

```diff
@@ -742,6 +742,9 @@ spec:
         displayName: Image
         path: image
       version: v1alpha1
+    - kind: KogitoTrusty
+      name: kogitotrusty.app.kiegroup.org
+      version: v1alpha1
```

Comment:
We are missing the CSV description of each component that makes part of this type. Have you run `make test` or `make vet`?

Reply:
Should be fine now, could you please double check?